orchagent/routeorch.h

```diff
@@ -84,6 +87,9 @@ class RouteOrch : public Orch, public Subject
     bool addRoute(IpPrefix, IpAddresses);
     bool removeRoute(IpPrefix);
 
+    std::string getLinkLocalEui64Addr(void);
```

Comment:
where is this function called?

Reply:
This is now called in routeorch initializing code. Please check the latest code.

Reply:
@jipanyang Could you please review the latest changes?